packages/flutter/test/cupertino/text_field_test.dart

```diff
@@ -435,6 +435,31 @@ void main() {
     },
   );
 
+  testWidgets(
+    'placeholder textAlign is equal to inputText',
```

Comment:
nit:
```suggestion
    'placeholder respects textAlign'
```

Reply:
@HaijunWei could you take a look at this line?

Reply:
sorry. Did not notice the suggestion.
I agreed to it, but some checks failed, what should I do?

Reply:
Whoops sorry forgot to put the trailing comma. Could you add the comma back?

Reply:
```suggestion
    'placeholder respects textAlign',
```

Reply:
@LongCatIsLooong   Ok. I added.